src/modules/mentors/mentors.controller.ts

```diff
@@ -122,9 +131,25 @@ export class MentorsController {
       roles: [...user.roles, Role.MENTOR],
     });
     
-    this.usersService.update(userDto);
+    
     const res: any = await this.mentorsService.updateApplication(applicationDto);
 
+    this.usersService.update(userDto);
+
+    let templateId = null
+    if (applicationDto.status === Status.REJECTED) {
+      templateId = EmailService.TEMPLATE_IDS.MENTOR_APPLICATION_REJECTED
+    } else {
+      EmailService.TEMPLATE_IDS.MENTOR_APPLICATION_APPROVED
```

Comment:
Wondering if omitting the assignation was on purpose?? 

```suggestion
templateId = EmailService.TEMPLATE_IDS.MENTOR_APPLICATION_APPROVED
```

Reply:
Nope, not on purpose.  It was the result of hasty, late night coding 🙃